Apps/W1/ReviewGLEntries/app/src/tables/GLEntryReviewLog.Table.al

```diff
@@ -0,0 +1,49 @@
+namespace Microsoft.Finance.GeneralLedger.Review;
+
+table 22218 "G/L Entry Review Log"
+{
+    Caption = 'G/L Entry Review Log';
+    DrillDownPageId = "Reviewed G/L Entries";
+
+    fields
+    {
+        field(1; "Line No."; Integer)
+        {
+            DataClassification = SystemMetadata;
+            AutoIncrement = true;
+        }
+        field(2; "G/L Entry No."; Integer)
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(3; "Reviewed Identifier"; Integer)
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(4; "Reviewed By"; Code[50])
+        {
+            DataClassification = EndUserIdentifiableInformation;
+        }
+        field(5; "Reviewed Amount"; Decimal)
+        {
+            DataClassification = EndUserIdentifiableInformation;
+        }
+        field(6; "G/L Account No."; Code[20])
+        {
+            Caption = 'G/L Account No.';
+            DataClassification = EndUserIdentifiableInformation;
```

Comment:
The DataClassification maybe should be check again.
Normally i would expected something like `CustomerContent` for fields like the `G/L Account No.` or the `Reviewed Amount`